omnigibson/utils/deprecated_utils.py

```diff
@@ -286,6 +290,88 @@ def get_joint_limits(
             values = self._backend_utils.convert(values, dtype="float32", device=self._device, indexed=True)
             return values
 
+    def get_joint_position_targets(
```

Comment:
can you add comments to the lines of code that you modify from the original source code?